static/js/admin.js

```diff
@@ -759,13 +765,20 @@ function _setup_page() {
                 full_name: JSON.stringify(full_name.val()),
             };
 
+            if (owner.val() !== undefined && owner.val() !== "") {
+                url = "/json/bots/" + email;
```

Comment:
> Should we be using encodeURIComponent here?

Yeah that's a mistake, will correct it.

>  Also, what happens here if both full_name and owner get set when you hit the submit button for the form?

It will work correctly in that case as well. Both the endpoints, `/bots/<bot_email>` as well as `/users/<bot_email>`, support changing bot's full name. I should have used only `/bots/<bot_email>`. Will correct it.